Firestore/core/src/firebase/firestore/local/local_documents_view.h

```diff
@@ -103,6 +103,18 @@ class LocalDocumentsView {
   /** Queries the remote documents and overlays mutations. */
   model::DocumentMap GetDocumentsMatchingCollectionQuery(FSTQuery* query);
 
+  /** It is possible that a `PatchMutation` can make a document match a query,
```

Comment:
Nit: in a multiline comment, `/**` should be on its own line:
```
/**
 * It is possible that a `PatchMutation` can make a document match a query,
```

Reply:
Done.